tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"program":{"fileNames":[],"fileInfos":[],"root":[],"options":{"declaration":true,"esModuleInterop":true,"inlineSourceMap":true,"module":1,"removeComments":false,"strict":true,"target":4,"useUnknownInCatchVariables":false},"referencedMap":[],"exportedModulesMap":[]},"version":"5.1.6"}
```

Comment:
@rwat17 This looks like a build artifact that somehow escaped git ignore. If I'm wrong, could you please explain why we need this in the diff?

Reply:
sorry I forgot to delete this :)